### PR TITLE
Qta 694

### PR DESCRIPTION
**Pull request by alecBlantonMoxiworks**
Add a tests to cover performance for ENG-175719

### CODE REVIEW

File: wms_svc_profile/locust_eng175719.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    @task()
+    def get_profile_v2_syndication_search(self):
+        try:
+            resp = self.client.get(url=self.syndication_profile_endpoint,
+                                   params={**self.syndication_prof_params, **self.auth})
+            assert resp.status_code == 200
+        except Exception as e:
+            print("request_failed", e)
+
+    # @task()
```
Comment (alecBlantonMoxiworks) [inline wms_svc_profile/locust_eng175719.py]: endpoint times out, leaving commented for now.

File: wms_svc_profile/locust_eng175719.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    # def get_profile_v2_attr_search(self):
+    #     try:
+    #         resp = self.client.get(url=self.v2_profile_attr_search_endpoint,
+    #                                params={**self.attr_search_params, **self.auth})
+    #         assert resp.status_code == 200
+    #     except Exception as e:
+    #         print("request_failed", e)
```
Comment (aettobi) [inline wms_svc_profile/locust_eng175719.py]: dead code

**Review by aettobi — CHANGES_REQUESTED** [review]

File: wms_svc_profile/locust_eng175719.py (hunk 1 above)
Comment (afukuyama) [reply wms_svc_profile/locust_eng175719.py]: Does this endpoint still time out?

Comment (alecBlantonMoxiworks) [reply wms_svc_profile/locust_eng175719.py]: yes, it is still consistently timing out

**Review by aettobi — APPROVED** [review]
:shipit:

**Pull request merged**